core/src/lib.rs

```diff
@@ -28,6 +28,7 @@ mod font;
 mod html;
 mod library;
 pub mod loader;
+mod percentage;
```

Comment:
Nit: let's name the module `percent` too

Reply:
Mooted: I'm naming it `types` instead and adding a `Degrees` type.